jabgui/src/main/resources/org/jabref/gui/fieldeditors/identifier/IdentifierEditor.fxml

```diff
@@ -10,6 +10,15 @@
 <fx:root xmlns:fx="http://javafx.com/fxml/1" type="HBox" xmlns="http://javafx.com/javafx/8.0.112"
          fx:controller="org.jabref.gui.fieldeditors.identifier.IdentifierEditor">
     <EditorTextField fx:id="textField" prefHeight="0.0" HBox.hgrow="ALWAYS"/>
+    <Button fx:id="shortenDOIButton" disable="${controller.viewModel.isInvalidIdentifier}"
+            onAction="#shortenID"
+            styleClass="icon-button"
+            managed="${controller.viewModel.canFetchBibliographyInformationById}"
+            visible="${controller.viewModel.canFetchBibliographyInformationById}">
```

Comment:
This is wrong. We need to introduce a new field to control the state of the shortenDOI button. Then we need to modify the configure() method in BaseIdentifierEditorViewModel accordingly.